ad-click/ad-click-impl/src/test/resources/maestro_tests/ad_click_detection_flows/1_-_y.js_heuristic;_no_ad_domain_param;_u3_param_included.yaml

```diff
@@ -26,27 +26,25 @@ appId: com.duckduckgo.mobile.android
 - assertVisible:
     text: "Publisher site"
 - tapOn:
-    id: "com.duckduckgo.mobile.android:id/privacyGradeButton"
+    id: "com.duckduckgo.mobile.android:id/omnibarIconContainer"
 - assertVisible:
     text: "Requests Blocked from Loading"
 - tapOn:
     text: "Requests Blocked from Loading"
 - assertVisible:
-    text: "The following third-party domains' requests were blocked from loading because they were identified as tracking requests. If a company's requests are loaded, it can allow them to profile you."
+    text: "The following third-party domains’ requests were blocked from loading because they were identified as tracking requests. If a company's requests are loaded, it can allow them to profile you."
 - assertVisible:
     text: "About our Web Tracking Protections"
 - assertVisible:
-    text: "Ad Company"
+    text: "www.ad-company.site"
```

Comment:
I see this was removed. We need to check all the below:
```
- assertVisible:
    text: ".*Ad Company"
- assertVisible:
    text: "convert.ad-company.site"
- assertVisible:
    text: "www.ad-company.site"

```

Reply:
Good catch!